src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Utilities/MediaQuery/BitMediaQueryDemo.razor.scss

```
@@ -0,0 +1,21 @@
+// container gives them a consistent gap instead of the <br /> runs the page would otherwise need - and keeps
+// the gap from collapsing to nothing when the queries of a group all fall out at once.
+.example-content {
+    gap: 0.5rem;
+    display: flex;
+    flex-flow: column;
+    align-items: flex-start;
+
+    // The class handed to a BitMediaQuery in the "Style & Class" example. It lands on a component, and the
+    // scope attribute of this stylesheet only lands on the plain elements written in this page, so it is
+    // reached through the container that carries it.
+    > ::deep .custom-class {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)